minLineLength: 100
gaussianKernelSize: 13

hsvColor:
  sThreshold: 95
  vThreshold: 130

erodeKernelSize:
  rows: 3
  cols: 3
dilateKernelSize:
  rows: 3
  cols: 11

centerRect:
  width: 5
  height: 10

depthLimit: 1.5
maxDepthDiff: 1.0
depthThreshold: 0.3

verticalLineGradDiff: 10.0

gradientThreshold: 2.0
betaThreshold: 3.0

linesThreshold: 2
